elements/lisk-codec/src/string.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import { writeVarInt } from './varint';
+
+interface SchemaProperty {
+	readonly dataType: string;
+}
+
+export const writeString = (value: string, _schema: SchemaProperty): Buffer =>
+	Buffer.concat([
+		Buffer.from('0a', 'hex'),
```

Comment:
do we need this?

Reply:
I think this `key` is not used now in other writers

Reply:
in order to produce hex as in protocol-spec, I had to add this line separator, for example, `lisk` -> `0a046c69736b` also @shuse2 mentioned it in the call to add it

Reply:
I think `0a` is the field number, and it changes based on the field number value

Reply:
Oh yes the key is calculated outside then we might need to revisit the protocol-spec and not use them for now

Reply:
So is it that, `writeString = (key: string, value: string, _schema: SchemaProperty): Buffer`, it takes a key as a string and concatenates as a buffer?

Reply:
`key` will be assigned from outside, so 
```
writeString = (value: string, _schema: SchemaProperty): Buffer
```
should return the `<length specifier><actual buffer>`?

Reply:
okay sounds good 👍